opentelemetry-exporter-cloud-monitoring/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -44,6 +47,8 @@
 
 # pylint is unable to resolve members of protobuf objects
 # pylint: disable=no-member
+# pylint: disable=too-many-branches
```

Comment:
Temporarily added. This file is overdue for some cleaning up, will do that after to avoid having a PR with refactor + logic changes.